pages/users-page.ts

```diff
@@ -0,0 +1,34 @@
+// Users page before installation
+
+export class UsersPage {
+  readonly page: Page;
+  readonly defineUserButton: Locator;
+  readonly setRootPwdDots: Locator;
+  readonly setRootPwdMenuItem: Locator;
+  readonly backButton: Locator;
+  readonly noUserDefinedTxt: Locator;
+  readonly rootPwdNotSetTxt: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.defineUserButton = page.getByRole('button', { name: 'Define a user now' });
+    this.setRootPwdDots =  page.getByRole('row', { name: 'Password Not set Actions' }).getByRole('button', { name: 'Actions' });
```

Comment:
s/setRootPwdDots/configureRootPasswordThreeDotsMenu/

Reply:
"define" is just a word the use for that particular screen, but the UI control is used everywhere to "change configuration"  of whatever item (this is the exact words the developers used in the demo), btw, I would prefer ThreeDots, as it is more easy to identify visually, and no need to learn new strange word.